src/annotation.ts

```diff
@@ -86,7 +86,7 @@ export class Annotation extends Modifier {
       const annotation = annotations[i];
       const textFormatter = TextFormatter.create(annotation.textFont);
       // Text height is expressed in fractional stave spaces.
-      const textLines = (5 + textFormatter.maxHeight) / Tables.STAVE_LINE_DISTANCE;
+      const textLines = (2 + textFormatter.getYForStringInPx(annotation.text).height) / Tables.STAVE_LINE_DISTANCE;
```

Comment:
Why did we move from max height of the font to height of the actual text? I think it is a good change for smufl fonts where some characters are very high (ie clefs) compared to actual text.

Reply:
My plan is to get rid of `maxSizeGlyph` as a member of the text metrics, and get the max height value directly from the font data.  Picking some arbitrary character seemed...arbitrary.  But when I used this value, it was higher than what we had been using, and the annotations took to much space.  And also people had complained about the much extra whitespace.  Using the size of the actual text tightens it up a little bit.

Reply:
Maybe for annotation we could add padding to options?  That would let you decide how much space goes between annotations, like for multiple verses.  Annotations above/below usually have to share space with articulations/ornaments so we want to use as little space as possible by default.